www/js/controllers/followgroup.js

```diff
@@ -0,0 +1,158 @@
+angular.module('app.controllers').controller('followgroup', function($scope, $state, $stateParams, $rootScope, flurry, followgroup, $http, serverConfig, follows, session, users, $ionicPopup){
+  var id = parseInt($stateParams.id, 10);
+  
+  $scope.loadGroupData = function(showSpinner){
+
+    $scope.data = [];
+    
+
+    if(showSpinner){
+      $scope.showSpinner();
+    }
+    $http.get(serverConfig.url + '/api/groups/' + id + '/users').then(
+      function (response) {
+        $scope.groupData = response.data;
+        member_count = $scope.groupData.length;
+        
+        followingData = follows.getFollowing();
+        following_count = followingData.length;
+        for (i = 0; i < member_count; i++){
+          var member = $scope.groupData[i];
+          var item = {id:"0", first_name:"", following:false, image_name:""};
+          item.id = member.id;
+          item.first_name = member.first_name;
+          item.username = member.username;
+          item.following = false;
+
+          for (j = 0; j < following_count; j++){
+            var following = followingData[j];
+            if (member.id == following.attributes.user.id){
+              item.following = true;
+              break;
+            }
+          }
+          $scope.data.push(item);
+        }
+
+        $scope.hideSpinner();
+      }
+    );
+  }
+
+  $scope.follow = function(item){
+
+    var following = follows.getByUserId(item.id);
+    if (item.following == true){
+      following.unfollow();
+    }
+    else{
+      following.follow();
+    }
+    item.following = !(item.following);
+  };
+
+  $scope.followAll = function(){
+    var confirmPopup = $ionicPopup.confirm({
+     title: 'Follow Every Member',
+     template: 'Send follow request to every group member? (Not recommended!)'

```

Comment:
At the beginning Jesse give me the requirement like that.
Follow All Members is not recommended to user.